BoldGrid/weforms
# Payment issue 70

@bwmarkle Code provided from mahesh prior to departure that fixed the transaction table issue. Also included emogrifier updates. Made sure that the code does not disrupt previous/pending merges with regards to weSerialize and PHP8 fixes (emogrifier regex update). 

To test broken state:
SCRIPT_DEBUG must be enabled to use the non-minified file.

From the master branch:

Create a payment form and proceed through the payment process.
![image](https://user-images.githubusercontent.com/25781560/119023156-b3d3d200-b95e-11eb-96fc-4728bebadcea.png)

** Note fix to frontend-form.js is added (weSerialize) or the form would not submit. **

- Payment information
email: elanad@inmotionhosting.com
card: 4242 4242 4242 4242
exp date: 04/24
cvc: 424
name: 424242424
zip: 90245

The transaction table id shows as zero instead of the entry id created in the entries table.

To Test Fix:

SCRIPT_DEBUG must be enabled to use the non-minified file.

From the Payment issue 70 branch:

Use the form created above and submit the entry and payment.

You will see that the transaction table id matches the entry id created
